### PR TITLE
Ai server improvement

### PR DESCRIPTION
**Pull request by aarondr77**
# Description

Adds some very very basic telemetry, just so we can see if the ai server is working or not. It either logs a `success` message or an `error` message. 

# Documentation

We should make sure the privacy policy is clear and accessible to users.

### CODE REVIEW

File: mito-ai/mito-ai/utils/open_ai_utils.py [hunk 1]
```diff
@@ -9,11 +9,18 @@
...
+from .telemetry_utils import log
 OPEN_AI_URL = 'https://api.openai.com/v1/chat/completions'
 MITO_AI_URL = 'https://ogtzairktg.execute-api.us-east-1.amazonaws.com/Prod/completions/'
 
+KEY_TYPE_PARAM = 'key_type'
+MITO_SERVER_KEY= 'mito_server'
+USER_KEY = 'user_key'
+
+MITO_AI_COMPLETION_SUCCESS = 'mito_ai_completion_success'
+MITO_AI_COMPLETION_ERROR = 'mito_ai_completion_error'
```
Comment (aarondr77) [inline mito-ai/mito-ai/utils/open_ai_utils.py:22]: Maybe these should all be moved to the telemetry_utils file?

**Review by aarondr77 — COMMENTED** [review]
small cleanup

**Comment by aarondr77** [comment]
Continue review test

**Review by aarondr77 — COMMENTED** [review]
test

**Review by ngafar — APPROVED** [review]

**Pull request closed without merging**